plugins/aws/resoto_plugin_aws/resource/elasticache.py

```diff
@@ -456,6 +462,8 @@ def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
                 clazz=AwsElastiCacheCacheCluster,
                 name=cluster_name,
             )
+        if self.replication_group_kms_key_id:
+            builder.dependant_node(self, reverse=True, clazz=AwsKmsKey, id=self.replication_group_kms_key_id)
```

Comment:
Shouldn't the KMSKey a successor instead of predecessor?

Reply:
Yes - copy & paste error, my bad!